homeassistant/components/fritz/common.py

```diff
@@ -146,17 +157,17 @@ def scan_devices(self, now: datetime | None = None) -> None:
             if not known_host.get("mac"):
                 continue
 
-            dev_mac = known_host["mac"]
-            dev_name = known_host["name"]
-            dev_ip = known_host["ip"]
-            dev_home = known_host["status"]
+            dev_mac: str = known_host["mac"]
+            dev_name: str = known_host["name"]
+            dev_ip: str = known_host["ip"]
+            dev_home: bool = known_host["status"]
```

Comment:
With `TypedDict` this should be no longer needed.
```suggestion
            dev_mac = known_host["mac"]
            dev_name = known_host["name"]
            dev_ip = known_host["ip"]
            dev_home = known_host["status"]
```